pkg/config/config.go

```diff
@@ -281,6 +281,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("fips.enabled", false)
 	config.BindEnvAndSetDefault("fips.port_range_start", 3833)
 	config.BindEnvAndSetDefault("fips.local_address", "localhost")
+	config.BindEnvAndSetDefault("fips.https", true)
```

Comment:
Plenty of other config use `ssl` instead of `https`. I think `fips.ssl` would be a more consistent name, WDYT?

Reply:
I don't really like using SSL, it's an outdated term. I don't have a super strong opinion, as consistency could be more important, but that's the reason why I went for `https` and why I somewhat favor it.